scripts.js

```diff
@@ -20,6 +20,8 @@ state = parseState(); // Map State Parsed from POST Arguments (Marker Visibility
 lastParamUpdate = 0; // Last Time POST Arguemts were updated
 
 
+var CURRENT_YEAR = 2019;
```

Comment:
Make this dynamically calculated.

Reply:
Since I know you won't want to update it exactly after the year starts, make it also check the month and then flip to the next year after October or something.